packages/hydrojudge/src/utils.ts

```diff
@@ -1,23 +1,32 @@
 import path from 'path';
 import { parse } from 'shell-quote';
-import { fs, sleep } from '@hydrooj/utils';
+import { fs } from '@hydrooj/utils';
 import { FormatError } from './error';
 
 const EMPTY_STR = /^[ \r\n\t]*$/i;
 
 export const cmd = parse;
 
 export namespace Lock {
-    const data = {};
+    const data: Record<string, Array<Function>> = {};
```

Comment:
`Record<string, Array<() => void>>`, also maybe call it a queue?

Reply:
I'll add `?` after `res` since this promise does not actually return any value.